docs/status.md

```diff
@@ -135,7 +135,7 @@ one of the following tracking labels.
 | TanhOp |yes|yes*|yes*|yes|yes|
 | TorchIndexSelectOp |no|no|no|no|no|
 | TraceOp |no|yes*|no|yes|no|
-| TransposeOp |no|yes*|yes*|no|no|
+| TransposeOp |yes|yes|yes*|no|yes|
```

Comment:
Is it intended for Type Inference to stay at "yes*" here? At a glance, the spec is already implemented correctly: https://github.com/openxla/stablehlo/blob/be42540a288a302b6e4071e7d29c0bf8b911589e/stablehlo/dialect/StablehloOps.cpp#L4983-L5022.

Reply:
I agree. Updated.